azure-pipelines.yml

```diff
@@ -111,7 +111,9 @@ jobs:
     vmImage: 'vs2017-win2016'
   timeoutInMinutes: 5
   steps:
-    - script: dotnet run --project ./src/dotnet-format.csproj -- @validate.rsp
+    - script: dotnet publish ./src/dotnet-format.csproj -c Release
+      displayName: Publish dotnet-format
+    - script: dotnet ./artifacts/bin/dotnet-format/Release/netcoreapp2.1/publish/dotnet-format.dll @validate.rsp
```

Comment:
In order to run codestyle analyzers, we need to publish before we run because not all dependencies are copied into the build folder.